src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj

```diff
@@ -6,23 +6,24 @@
     <Description>NuGet's configuration settings implementation.</Description>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
```

Comment:
Did you consider compiling everything into 1 assembly?

That we don't pay the cost of per tfm analyzer + the additional tfms beign built.

You'd have to build only 1 assembly.

Reply:
didn't consider. will noodle.

Reply:
tracking that possibility in https://github.com/nuget/client.engineering/issues/461